src/lib/es6.d.ts

```diff
@@ -127,7 +127,7 @@ interface ObjectConstructor {
       * @param target The target object to copy to.
       * @param source The source object from which to copy properties.
       */
-    assign<T, U>(target: T, source: U): T & U;
+    assign<T, U>(target: T, source: U | null | undefined): T & U;

```

Comment:
Actually, on closer look it seems U is still specialized as `null` instead of `{}`. But also the resulting type T & null doesn't seem to have any issues either.

Should I remove the `| null | undefined` from this function?